internal/pkg/completion/completion.go

```diff
@@ -0,0 +1,119 @@
+package completion
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/devstream-io/devstream/cmd/devstream/list"
+	"github.com/devstream-io/devstream/pkg/util/log"
+)
+
+func ArgsPluginsCompletion(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
> I don't think this is the problem. When the user wants to know how many plugins are related to `argocd`, they can simply do `dtm list plugins argocd[TAB][TAB]` to filter them out. I believe this is also the purpose of `dtm list plugins -r`.

I agree that we need to allow users to filter plugins. So `dtm list plugins -r` should be accepted. However, in my opinion, the premise `dtm list plugins argocd[TAB][TAB]` can be accepted is that `dtm list plugins` accepts the plugin name as arg or subcommand. I don't know if you understand what I mean.

 I think `dtm list plugins -r xxx` is good. The result can be used as `stdin` for other commands. For example
```
dtm list plugins -r "\bargocd\b" | dtm show config
dtm show config $(dtm list plugins -r "\bargocd\b")
```

The results of `dtm list plugins argocd`  are the same as `dtm list plugins`.  In addition, we have to clean up the command manually before executing the next command.

In summary, why do we need `dtm list plugins argocd[TAB][TAB]`. 

What do you think? @daniel-hutao @IronCore864

Reply:
Right, the `dtm list plugins argocd` command does not exist; `dtm list plugins` is used to print all plugins, and this command has no further subcommands, it only has flags, like `-r`. So `dtm list plugins [tag]` should not output specific plugin names.